#!/usr/bin/python3
if __name__ == "__main__":
    def raise_exception():
        raise TypeError("Exception has been raised")

